cost/volumes/unattached_volumes/uav_policy.pt

```diff
@@ -337,4 +370,17 @@ define handle_delete_failure(@volume) do
       rs_cm.tags.multi_add(tags: [$tag], resource_hrefs: [@volume.href])
     end
   end
+  $summary = join(["delete-error-",@volume.name])
+  call sys_log($summary,$msg)
 end
+
+define sys_log($subject, $detail) do
+  rs_cm.audit_entries.create(
+    notify: "None",
+    audit_entry: {
+      auditee_href: @@account,
+      summary: $subject,
```

Comment:
include the policy name in the subject so it's easy to find in the audit entries.